### PR TITLE
fix: Remove Hashtag From Route

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
 * setting `mode: 'history'` to avoid # in routes (see Todo)
 * added some missing locales (members area)
 * changed locale of members area
 * removed use of store in Send compoenent

### Issues
- relates #104

### Todo

- [ ] On page load there can be seen a 404 for a short time. After that, everything runs fine. I think it has to do with the nginx config (see https://router.vuejs.org/guide/essentials/history-mode.html#nginx)

### CODE REVIEW

**Review by ulfgebhardt — COMMENTED** [review]
I do not really understand how this addresses the hashtag thing? Mind explaining please?

**Comment by Mogge** [comment]
We need to have the `mode: 'history'` in the router setup.

File: frontend/src/views/Layout/DashboardLayout_gdd.spec.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ describe('DashboardLayoutGdd', () => {
       })
 
       it('has first item "send" in navbar', () => {
-        expect(navbar.findAll('ul > li').at(0).text()).toEqual('sent')
+        expect(navbar.findAll('ul > li').at(0).text()).toEqual('send')
```
Comment (Mogge) [inline frontend/src/views/Layout/DashboardLayout_gdd.spec.js:86]: How did this get into master?

Comment (ulfgebhardt) [reply frontend/src/views/Layout/DashboardLayout_gdd.spec.js:86]: The Test is not yet required, fix it and we can make that happen

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**